Ten Elephants/Ten Elephants/Models/Meals.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  Meals.swift
+//  Ten Elephants
+//
+//  Created by Дарья Домрачева on 09.12.2021.
+//
+
+import Foundation
+
+struct  Meals: Decodable {
+    enum CodingKeys: String, CodingKey {
+        case meals
+    }
+    let meals: [MealModel]
+}
+
+struct MealModel: Decodable {
+    // always present
+    let name: String               // strMeal
+    let id: String                 // idMeal
+    let thumbnailLink: String      // strMealThumb
+
+    // might be present in full model
+    let category: String?          // strCategory
+    let area: String?              // strArea
+    let instructions: String?      // strInstructions
+    let youTubeLink: String?       // strYoutube
+    let tags: [String]?            // strTags
+
+    let ingredients: [Ingredient]? // Ingredient(strIngredient, strMeasure)
+
+    enum RequiredCodingKeys: String, CodingKey, CaseIterable {
+        case strMeal,
+             idMeal,
```

Comment:
Предложу явно писать все кейсы, так обычно меньше диффа в плане расставке запятых
```suggestion
         case idMeal
```